README.md

```diff
@@ -0,0 +1,65 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project 
+
+# Community Development 
+
+![screenshot](img/ScreenCap.png)
+
+>This website if a community development website dedicated for my local city
+
+>
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+
+## Live presentation
+
+https://drive.google.com/file/d/1xbWiByBpvsrHnwA5M1SkdVV5oY7w40aV/view?usp=sharing
```

Comment:
- [ ] It isn't possible to view the video on google drive as there is file permission restriction, please ensure you set the video to be viewed by the public and ensure that you can watch the video on an incognito browser to ensure it can be viewed by all.

Reply:
thanks for that but i am having loom problems so thats why i used google drive do you have any suggestions on a better video recorder because i just fear i am left with one review